zerver/lib/markdown/help_settings_links.py

```diff
@@ -114,16 +114,32 @@
 1. On the left, click {setting_reference}.
 """
 
+users_tab_markdown = """
+1. Click on the **gear** (<i class="zulip-icon zulip-icon-gear"></i>) icon in the upper
+   right corner of the web or desktop app.
+
+1. Select **{setting_type_name}**.
+
+1. On the left, click **Users**.
+
+1. Select the {setting_reference} tab.
```

Comment:
It is nice to have this step automatically generated here and not present in the relative settings link case. It just feels like we need three variables - the setting type (org/personal), the setting tab (users) and the setting reference - for this to work with the relative link case as well.